controllers/storagecluster/external_resources.go

```diff
@@ -326,6 +344,82 @@ func (obj *ocsExternalResources) ensureDeleted(r *StorageClusterReconciler, inst
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageClusterReconciler) createStorageClassClaim(instance *ocsv1.StorageCluster) error {
+
+	for _, desiredStorageClassClaim := range getDefaultStorageClassClaims(instance) {
+
+		err := controllerutil.SetOwnerReference(instance, desiredStorageClassClaim, r.Client.Scheme())
+		if err != nil {
+			return err
+		}
+
+		err = r.Client.Create(context.TODO(), desiredStorageClassClaim)
+		if err != nil && !errors.IsAlreadyExists(err) {
+			return err
+		}
+
+	}
+
+	return nil
+}
+
+func (r *StorageClusterReconciler) deleteStorageClassClaim(instance *ocsv1.StorageCluster) error {
+
+	for _, desiredStorageClassClaim := range getDefaultStorageClassClaims(instance) {
+		err := r.Client.Delete(context.TODO(), desiredStorageClassClaim)
+
+		if err != nil && !errors.IsNotFound(err) {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (r *StorageClusterReconciler) verifyNoStorageClassClaimsExist(instance *ocsv1.StorageCluster) error {
+
+	storageClassClaims := &ocsv1alpha1.StorageClassClaimList{}
+	err := r.Client.List(context.TODO(), storageClassClaims, &client.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	if len(storageClassClaims.Items) != 0 {
+		err = fmt.Errorf("Failed to cleanup resources. storageClassClaims are present." +
```

Comment:
use errors.New when no error formatting is required.

Reply:
done